composer.json

```diff
@@ -31,7 +31,7 @@
         "ext-phar": "*",
         "composer-runtime-api": "^2.0.0",
         "nikic/php-parser": "^4.13.0",
-        "symfony/console": "^5.4.0",
+        "symfony/console": "^5.4.0 || ^6.0.0",
```

Comment:
IMO just drop 5.4.x here: this is an application, not a library.

Reply:
👍🏾